--- /Users/Kevin/GitHub/TailscaleGX.copy/FileSets/PatchSource/PageSettingsGeneral.qml.orig	2024-04-22 04:01:13
+++ /Users/Kevin/GitHub/TailscaleGX.copy/FileSets/PatchSource/PageSettingsGeneral.qml	2024-05-02 17:31:26
@@ -80,6 +80,13 @@
 			show: remoteSupportOnOff.item.value
 		}
 
+////// added for tailscale remote connections
+		MbSubMenu
+		{
+			description: qsTr("Remote access via tailscale")
+			subpage: Component { PageSettingsTailscaleGX {} }
+		}
+
 		MbOK {
 			id: reboot
 			description: qsTr("Reboot?")
