--- /Users/Kevin/GitHub/TailscaleGX.copy/FileSets/PatchSource/PageSettingsGeneral.qml-1.orig	2024-02-20 09:46:56
+++ /Users/Kevin/GitHub/TailscaleGX.copy/FileSets/PatchSource/PageSettingsGeneral.qml-1	2024-03-29 18:10:29
@@ -80,6 +80,13 @@
 			show: remoteSupportOnOff.item.value
 		}
 
+////// added for tailscale remote connections
+		MbSubMenu
+		{
+			description: qsTr("Remote access via tailscale")
+			subpage: Component { PageSettingsTailscaleGX {} }
+		}
+
 		MbOK {
 			id: reboot
 			description: qsTr("Reboot?")
